src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java

```diff
@@ -382,6 +390,14 @@ public boolean equals(Object obj) {
             return false;
         }
 
+        if (questionDescription == null ^ other.questionDescription == null) {
```

Comment:
I just noticed that the previous way is copied from `questionMetaData` which is directly above this. In that case, I think we should revert to the previous way to make it standardized. Sorry about this.